src/utilities/categoryFilterHandler.js

```diff
@@ -0,0 +1,13 @@
+const categoryFilterHandler = (products, categories) => {
+  return products.filter((product) => {
+    for (let i = 0; i < categories.length; i++) {
```

Comment:
you can use map here

Reply:
Sure will do it in next PR. For the time being I've created an [issue](https://github.com/VishalPatil18/viskart-react-version/issues/16) for the same.